<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>courseReservation</class>
 <widget class="QWidget" name="courseReservation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>课程预约</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="filterGroupBox">
     <property name="title">
      <string>筛选条件</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="startDateLabel">
        <property name="text">
         <string>开始日期:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDateEdit" name="startDateEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="endDateLabel">
        <property name="text">
         <string>结束日期:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDateEdit" name="endDateEdit"/>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="filterButton">
        <property name="text">
         <string>筛选</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="courseTableView"/>
   </item>
   <item>
    <widget class="QGroupBox" name="reservationGroupBox">
     <property name="title">
      <string>预约操作</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="reserveButton">
        <property name="text">
         <string>预约</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="text">
         <string>取消预约</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="myReservationTableView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>